.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "extends": [
```

Comment:
Missing an ending newline on these two JSON files, and their format differs from the others (spaces)